<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>imgproc</string>
  </property>
  <property name="windowIcon">
   <iconset resource="mainwindow.qrc">
    <normaloff>:/MainWindow/Resources/imgproc.png</normaloff>:/MainWindow/Resources/imgproc.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QtOcv::ImageWidget" name="originalView"/>
      <widget class="QtOcv::ImageWidget" name="processView"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="actionRecent_Files"/>
    <addaction name="separator"/>
    <addaction name="action_Save"/>
    <addaction name="actionSave_As"/>
   </widget>
   <widget class="QMenu" name="menuProcess">
    <property name="title">
     <string>&amp;Process</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuProcess"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Open"/>
   <addaction name="action_Save"/>
   <addaction name="actionSave_As"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="filterDockWidget">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Image Filter</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWidget" name="paramsWidget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_2"/>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnApply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSetBase">
       <property name="text">
        <string>SetBase</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/MainWindow/Resources/open.png</normaloff>:/MainWindow/Resources/open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/MainWindow/Resources/save.png</normaloff>:/MainWindow/Resources/save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/MainWindow/Resources/saveas.png</normaloff>:/MainWindow/Resources/saveas.png</iconset>
   </property>
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
  </action>
  <action name="actionRecent_Files">
   <property name="text">
    <string>Recent Files</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QtOcv::ImageWidget</class>
   <extends>QGraphicsView</extends>
   <header>cvimagewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
